internal/rbd/manager.go

```diff
@@ -0,0 +1,89 @@
+/*
+Copyright 2024 The Ceph-CSI Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package rbd
+
+import (
+	"context"
+	"errors"
+
+	types "github.com/ceph/ceph-csi/internal/rbd_types"
```

Comment:
can we move this to a folder inside the `internal/rbd`? like `internal/rbd/types`?

Reply:
That was not possible earlier because it would import the `internal/rbd` package, and `internal/rbd` imports the types too. It's been cleaned up quite a bit since, and maybe it is possible now.